components/checkbox/src/index.js

```diff
@@ -88,23 +88,25 @@ const StyledLabel = styled('span')({
 });
 
 const Checkbox = ({
-  children, className, inline, ...input
+  children, inline, disabled, defaultChecked, ...props
 }) => (
-  <StyledCheckbox className={className} inline={inline}>
-    <StyledInput type="checkbox" {...input} />
+  <StyledCheckbox inline={inline} {...props}>
```

Comment:
i don't think that we should have {...props} here but instead...

Reply:
however this may well be the place that we want to put any `className` that may have been passed in...